### ISSUE TITLE
TW Day2 11:45〜 "Draw me like one of your French Girls. — Generative models in Python"

### ISSUE THREAD

**Issue by ftnext** [ftnext/PyConTalkSummary#11]
## 一言でいうと

生成モデルについてGANを中心に紹介したトーク。
サーベイ論文のように、用語の整理、生成モデルの位置づけ、GANの説明、GANの事例の紹介

### 発表資料リンク

- 発表資料：[TODO: ここにリンクを入れる]
- YouTube：[TODO: ここにリンクを入れる]
- トーク詳細：https://tw.pycon.org/2019/en-us/events/talk/871233384901247285/

### 発表者/所属

[Prakhar Srivastava](http://prsr.me/)

### 発表日付

2019/09/21

## 概要

Unsupervised learning
（学習時に正解とするラベルがない）

### types of unsupervised learning

- Clustering
- dimensionary reduction
- density estimation 密度推定 ref: https://scikit-learn.org/stable/modules/density.html

Generative modelsも教師なし学習の1つ
training dataと同じdataを作ろうとする
extrapolate（外挿）→Imagination

### taxonomy of generative model

Morkov Chainもgenerative modelに含まれる

- explicit density
    - Pixel RNN/CNN
    - VAE（Variational AutoEncoders）
- implicit density
    - GANs

### GANs

implicit
2 network
both try to beat each other

- generator：RealWorld Imageに似せた画像を作る
- discriminator -> Loss　本物か偽物か見分ける

loss関数はmin max game

### further GANs

- Self-Attention GAN
- CycleGAN / Pix2pix

前者：https://arxiv.org/pdf/1805.08318.pdf (Figure 2)
正則化されたdiscriminator

pix2pix
styleを適用する
https://arxiv.org/pdf/1611.07004.pdf
馬に縞をつけてシマウマにしてしまう例

---

[以下はオプション]

### リソース類

TensorFlowのチュートリアルにある
MNISTの例
pix2pixの例

GANのチュートリアルビデオ
https://sites.google.com/view/cvpr2018tutorialongans/

image2image
https://affinelayer.com/pixsrv/

## 感想

「GAN=generative model」ではないことがわかった（生成モデルの1つで他にマルコフ連鎖などがある）
